File: ts/models/OrderLineModel.ts
import {EventModel} from "./EventModel";
import {TicketModel} from "./TicketModel";
export class OrderLineModel {
    public event:EventModel;
    public tickets:TicketModel[];

    public getCost():number{
        var sum:number=0;

        if(this.tickets!=null) {
            for(var i=0;i<this.tickets.length;i++) {
                sum+=this.tickets[i].cost;
            }
        }

        return sum;
    }
}

File: src/main/java/com/columbustheater/viewmodels/OrderLineModel.java
package com.columbustheater.viewmodels;

import com.columbustheater.models.Event;

import java.math.BigInteger;
import java.util.List;

public class OrderLineModel {
    private Event event;
    private List<TicketModel> tickets;

    public Event getEvent() {
        return event;
    }

    public void setEvent(Event event) {
        this.event = event;
    }

    public List<TicketModel> getTickets() {
        return tickets;
    }

    public void setTickets(List<TicketModel> tickets) {
        this.tickets = tickets;
    }
}
